sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/LossOfFractionInDivision.CSharp9.cs

```diff
@@ -0,0 +1,17 @@
+﻿decimal dec = 3 / 2;    // Noncompliant
+dec = 3L / 2;           // Noncompliant
+Method(3 / 2);          // Noncompliant
+dec = (decimal)3 / 2;
+Method(3.0F / 2);
+
+decimal ndec = (nint)3 / 2; // FN
```

Comment:
I think we should add cases when both operands are nint or nuint.
something like:
```
nint a = 2;
nint b = 4;
decimal x = b/a;
```